core/arch/arm/kernel/virtualization.c

```diff
@@ -140,7 +140,7 @@ void virt_init_memory(struct tee_mmap_region *memory_map)
 
 	/* Init page pool that covers all secure RAM */
 	if (!tee_mm_init(&virt_mapper_pool, TEE_RAM_START,
-			 TA_RAM_START + TA_RAM_SIZE,
+			 TA_RAM_SIZE,
```

Comment:
That should be `TA_RAM_START + TA_RAM_SIZE - TEE_RAM_START`

Reply:
Thanks! The new patch fix this.